File: BE/Src/MIT.Fwk.Infrastructure/Entities/LegalTerm.cs
using JsonApiDotNetCore.Resources;
using JsonApiDotNetCore.Resources.Annotations;
using System;
using MIT.Fwk.Core.Attributes;

namespace MIT.Fwk.Infrastructure.Entities
{
    [SkipJwtAuthentication(JwtHttpMethod.GET)]
    public class LegalTerm : Identifiable<string>
    {


        [Attr] public string Title { get; set; }
        [Attr] public string Note { get; set; }

        [Attr] public bool Active { get; set; }

        [Attr] public string Code { get; set; }
        [Attr] public string Language { get; set; }
        [Attr] public string Version { get; set; }
        [Attr] public string Content { get; set; }
        [Attr] public DateTime? DataActivation { get; set; }


    }
}

File: FE/WEB/app/models/legal-term.js
import Model, { attr } from '@ember-data/model';

export default class LegalTermModel extends Model {
  @attr('string', { defaultValue: '' }) title; // varchar 250
  @attr('string', { defaultValue: '' }) note; // varchar max
  @attr('string', { defaultValue: '' }) code; // varchar 35
  @attr('string', { defaultValue: '' }) language; // varchar 2
  @attr('string', { defaultValue: '' }) content; // varchar max
  @attr('boolean', { defaultValue: false }) active;
  @attr('string', { defaultValue: '' }) version; // varchar 15
  @attr('date') dataActivation; // dateTime
}
